inference-engine/tests/functional/inference_engine/onnx_reader/model_support_tests.cpp

```diff
@@ -0,0 +1,72 @@
+﻿// Copyright (C) 2018-2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <gtest/gtest.h>
+#include <fstream>
+
+#include <ie_core.hpp>
+
+namespace {
+    std::string model_path(const char* model) {
+        std::string path = ONNX_TEST_MODELS;
+        path += "support_test/";
```

Comment:
What about windows paths?
Will it work if path will have mixed `/` and `\` ?

Reply:
The CI shows that it works on Windows too.